errors.go

```diff
@@ -77,7 +81,9 @@ func HandleExitCoder(err error) {
 	}
 
 	if exitErr, ok := err.(ExitCoder); ok {
-		if err.Error() != "" {
+		if _, ok := exitErr.(ErrorFormatter); ok {
```

Comment:
Im not so clear why there is this test on empty err message. Does people sends empty errors ? weird :x